ecms/src/main/java/org/exoplatform/platform/qa/ui/ecms/pageobject/DocumentManagement.java

```diff
@@ -128,7 +129,7 @@ public void goToPermissions() {
             $(ELEMENT_ACTIONBAR_MORE).click();
             ELEMENT_BUTTON_PERMISSION_IN_MORE_DROP_DOWN_MENU.click();
         } else {
-            ELEMENT_BUTTON_PERMISSION.click();
+            ELEMENT_BUTTON_PERMISSION.waitUntil(Condition.exist,timeout).click();
```

Comment:
```suggestion
            ELEMENT_BUTTON_PERMISSION.waitUntil(exist, timeout).click();
```
You can use static import, like for timeout.

Reply:
Adding/removing imports will not break anything, or you will see it during compilation, so don't be afraid to change them.